zerver/tests/test_auth_backends.py

```diff
@@ -2461,6 +2502,11 @@ def test_login_failure(self) -> None:
             response = self.client_post('/accounts/login/local/', data)
             self.assertRedirects(response, reverse('dev_not_supported'))
 
+    def test_dev_direct_production_config_error(self) -> None:
+        result = self.client_get("/config-error/dev")
+        self.assertEqual(result.status_code, 200)
+        self.assert_in_success_response(["DevAuthBackend"], result)
```

Comment:
Why do we put this specifically in the Google test class?

Reply:
It's in `TestDevAuthBackend`. You might've been mistaken because the code in between was hidden and there was google test class just above this in github while reviewing.

Reply:
Yeah, github confused me - look good then :slightly_smiling_face: